openapi/openapi.yaml

```diff
@@ -257,6 +305,98 @@ paths:
 
 components:
   schemas:
+
+    Collection:
+      title: 'Collection'
+      description: 'Ansible content collection'
+      type: object
+      properties:
+        name:
+          description: 'Collection name. Must be lower case containing
+                        only alphanumeric characters and underscores.'
+          type: string
+          maxLength: 64
+          readOnly: true
+        remote_id:
```

Comment:
Even if `remote_id` was in the schema (and it was removed as well as local Collection model) it's a part of internal implementation detail, that has nothing to do with public API and should not be exposed.

Reply:
I'll stub out the Collection model then.